policies/lib/all_attestations.rego

```diff
@@ -0,0 +1,10 @@
+package lib
+
+#
+# A list of all attestations
+#
+all_attestations = attestations {
+	attestations := [attestation |
+		attestation := {"data": data.attestations}
```

Comment:
The plural "attestations" makes it look like an array, but I think maybe it's just one attestation now.